testsuite/integration-arquillian/tests/base/pom.xml

```diff
@@ -894,6 +894,8 @@
                                 <keycloak.map.storage.connectionsJpa.user>${keycloak.map.storage.connectionsJpa.user}</keycloak.map.storage.connectionsJpa.user>
                                 <keycloak.map.storage.connectionsJpa.password>${keycloak.map.storage.connectionsJpa.password}</keycloak.map.storage.connectionsJpa.password>
 
+                                <keycloak.storage.connections.vendor>postgres</keycloak.storage.connections.vendor>
```

Comment:
`postgres` or `postgresql` ?

Reply:
`postgres`

It is an alias, while `postgresql` is a databaseKind. We need to use the alias as CocroachDB and PostgreSQL have the same databaseKind and we need to distinguish between them.

Reply:
Thanks for the explanation. I see the help message was changed, and it makes sense to me right now.